discount/src/main/java/com/kyriba/conference/discount/api/dto/DiscountPercentageDto.java

```diff
@@ -0,0 +1,20 @@
+package com.kyriba.conference.discount.api.dto;
+
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import lombok.Getter;
+
+import javax.validation.constraints.Max;
+import javax.validation.constraints.Min;
+
+/**
+ * @author Igor Lizura
+ */
+@ApiModel(value = "DiscountPercentage", description = "Discount updation request")
+@Getter
+public class DiscountPercentageDto {
```

Comment:
what is a difference between DiscountPercentageDto and DiscountDto? There is "percentage" in both.

Reply:
was renamed